integration-tests/provisioning/conftest.py

```
@@ -0,0 +1,132 @@
+    Returns
+    -------
+    Path
+        The tmp-rooted ``etc/repovec`` directory the patched helper
+        writes into. Tests use this to populate the simulated config
+        tree before invoking the helper.
+    """
+
+    return patched_helper.parent / "etc" / "repovec"
+
+
+@pytest.fixture()
```

Ruff (0.15.15)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)